From: Fryderyk Dziarmagowski <fdziarmagowski@gmail.com>
Date: Fri, 14 Oct 2011 21:41:43 +0000 (-0400)
Subject: Link libobt into libobrender, as it uses things from libobt (bug #5150)
X-Git-Url: http://git.openbox.org/?p=dana%2Fopenbox.git;a=commitdiff_plain;h=b1039d6142865e3141c831ce75cd9799ce3ceab7

Link libobt into libobrender, as it uses things from libobt (bug #5150)

libobrender uses ObtPaths stuff, so it needs to link in libobt.
---

Index: openbox-3.5.0/Makefile.am
===================================================================
--- openbox-3.5.0.orig/Makefile.am	2011-08-01 22:14:58.000000000 +0200
+++ openbox-3.5.0/Makefile.am	2012-06-05 07:03:32.000000000 +0200
@@ -79,6 +79,7 @@
 obrender_libobrender_la_LDFLAGS = \
 	-version-info $(RR_CURRENT):$(RR_REVISION):$(RR_AGE)
 obrender_libobrender_la_LIBADD = \
+	obt/libobt.la \
 	$(X_LIBS) \
 	$(PANGO_LIBS) \
 	$(GLIB_LIBS) \
